app/assets/stylesheets/calls.scss

```diff
@@ -1,55 +0,0 @@
-// Place all the styles related to the calls controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
-
-div.calls-layout {
-  padding: 30px 15px 30px 60px;
-}
-
-h4.calls-request {
-  font-size: 26px;
-}
-
-h4.calls-phone {
-  font-size: 34px;
-  font-weight: 900;
-}
-
-.calls-response {
-  font-size: 17px;
-  text-decoration: underline;
-}
-
-.new-call-entry {
-  font-size: 17px;
-  font-size: 100%;
-}
-
-#calls-btn {
-  font-size: 17px;
-  padding: 10px 40px;
-}
-
-.handle > span {
-  color: #ddd;
-  margin-left: 5px;
-  font-size: 1.25em;
-}
```

Comment:
looking at the refactor this styling wasn't migrated anywhere - is this 

> the base html elements 

you mention [here?](https://github.com/DCAFEngineering/dcaf_case_management/pull/2106/files#r550838411)

Reply:
oh, the `base html elements` is about the h4s on some of the call modal styling. Kinda overkill to specify the base element on those.

But you're keeping me honest here - the story of that handle class is that it's a holdover before we started using fontawesome icons (due to bootstrap 4 dropping their icon library). There isn't a handle element in a span now, they're all svgs, so that's a dead class.